src/main/java/game/domain/car/RacingGameCar.java

```diff
@@ -0,0 +1,33 @@
+package game.domain.car;
+
+import game.domain.RacingGameRule;
+import game.domain.car.Car;
+
+public class RacingGameCar extends Car {
+
+    public RacingGameCar() {
+    }
+
+    public RacingGameCar(String name) {
+        super(checkNameLength(name));
+    }
+
+    @Override
+    public void forward(RacingGameRule racingGameRule) {
+        if (racingGameRule.isForward(racingGameRule.pickRandomNumber())) {
+            forward(racingGameRule.forwardDistance());
+        }
+    }
+
+    public void forward(int number) {
+        location += number;
+    }
+
+    private static String checkNameLength(String name) {
+        if(name.length()>5){
```

Comment:
컨벤션 한번 확인 부탁 드립니다 🙏 
```suggestion
        if (name.length() > 5) {
```
그리고 `5`는 매직넘버이니 추출해주시면 
`RuntimeException`의 `message`에도 재활용이 가능할 것 같습니다 😄